lib/signalfx/signalflow/websocket.rb

```diff
@@ -17,9 +17,10 @@ class SignalFlowWebsocketTransport
   # A lower bound on the amount of time to wait for a computation to start
   COMPUTATION_START_TIMEOUT_SECONDS = 30
 
-  def initialize(api_token, stream_endpoint)
+  def initialize(api_token, stream_endpoint, logger = Logger.new(STDOUT, progname: "SignalFX"))
```

Comment:
I'm not super familiar with Ruby but is there any danger to doing this in the parameter list? I know this can be an anti-pattern in Python for example.